src/store/reducer.test.js

```diff
@@ -0,0 +1,42 @@
+import {
+  setCategories, setFilter, setRegions, setRestaurants,
+} from './actions';
+import reducer, { initialState } from './reducer';
+
+describe('reducer', () => {
+  it('returns initialState', () => {
+    expect(reducer(undefined, {})).toEqual(initialState);
+  });
+
+  context('when setRegion is dispatched', () => {
+    it('returns state with new regions', () => {
+      const regions = [{ id: 1, name: '서울' }];
+
+      expect(reducer(undefined, setRegions(regions)).regions).toEqual(regions);
```

Comment:
```suggestion
      const regions = [{ id: 1, name: '서울' }];
      
      const state = reducer(undefined, setRegions(regions));

      expect(state.regions).toEqual(regions);
```

중간값을 변수로 추출하면 더 이해하기 좋을 것 같아요. 왜냐하면 테스트 이름에 `state`가 들어갔기 때문이에요

Reply:
좋은것 같습니다!